jsapp/js/main.es6

```diff
@@ -15,8 +15,8 @@ import {csrfSafeMethod, currentLang} from 'utils';
 require('../scss/main.scss');
 import Modal from 'react-modal';
 
-// Tell moment library what is the app language
-moment.locale(currentLang());
+// Tell dayjs library which language the app is using
+dayjs.locale(currentLang());
```

Comment:
1. Do we need to do anything differently to make sure the locales actually load? 
    - > By default, Day.js comes with English locale **only**. If you need other locales, you can load them on demand. 
      > — [DayJS i18n — Changing Locale](https://day.js.org/docs/en/i18n/changing-locale#:~:text=By%20default%2C%20Day.js%20comes%20with%20English%20locale%20only.)
2. We call `dayjs.locale(currentLang())` here and in utils.ts. Is there something going on with imports (tree shaking or lazy loading) that dictates we do this more than once? 
    - (submissionsCountGraph implicitly depends on the global locale setting because it uses localized formatting functions.)

Reply:
1. Good catch! I've added 'locales.ts' to cover importing those files in one place. I'm not 100% sure that it's the ideal solution - we can't import them dynamically based on the backend response, so it means another place that means updating if we add another language to the kpi interface. That could be changed to importing all locales, with the downside that we'd be loading some locales unnecessarily.

2. Nope, I just wanted to be excessively careful, but you're right that it's unnecessary. I've left the call in `main.es6`.